pkgs/tools/typesetting/pdf2odt/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, makeWrapper, fetchFromGitHub
+, bc, coreutils, file, gawk, ghostscript, gnused, imagemagick, zip }:
+
+stdenv.mkDerivation rec {
+  version = "2014-12-17";
+  name = "pdf2odt-${version}";
+
+  src = fetchFromGitHub {
+    owner = "gutschke";
+    repo = "pdf2odt";
+    rev = "master";
+    sha256 = "14f9r5f0g6jzanl54jv86ls0frvspka1p9c8dy3fnriqpm584j0r";
+  };
+
+  dontStrip = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  path = lib.makeBinPath [
+    bc
+    coreutils
+    file
+    gawk
+    ghostscript
+    gnused
+    imagemagick
+    zip
+  ];
+
+  patches = [ ./use_mktemp.patch ];
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/doc
+
+    install -m0755 pdf2odt $out/bin/pdf2odt
+    ln -rs $out/bin/pdf2odt $out/bin/pdf2ods
+
+    install -m0644 README.md LICENSE -t $out/share/doc
+
+    substituteInPlace $out/bin/pdf2odt \
```

Comment:
Indeed, it's just my sloppiness. I had used ```substituteInPlace``` first before doing a proper patch indeed. Nice catch. No PRs after 23:00 for me going forward...